src/libraries/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -419,6 +636,38 @@ public async Task SendReceive_Concurrent_Success(Uri server)
             }
         }
 
+        [ConditionalFact(nameof(WebSocketsSupported))]
+        public async Task SendReceive_Concurrent_Success_Base()
+        {
+
+            await CreateEchoServerAsync(async uri =>
+            {
+                using (ClientWebSocket cws = await GetConnectedWebSocket(uri, TimeOutMilliseconds, _output))
```

Comment:
```suggestion
                using ClientWebSocket cws = await GetConnectedWebSocket(uri, TimeOutMilliseconds, _output);
```
Lets you remove a set of braces here